include/pokemon.h

```diff
@@ -658,8 +658,8 @@ void SetMonMoveSlot(struct Pokemon *mon, u16 move, u8 slot);
 void SetBattleMonMoveSlot(struct BattlePokemon *mon, u16 move, u8 slot);
 void GiveMonInitialMoveset(struct Pokemon *mon);
 void GiveBoxMonInitialMoveset(struct BoxPokemon *boxMon);
```

Comment:
Aren't these two lines redundant now?

Reply:
They are. Interesting that it still compiles. Removed them.